docs/draft-alakuijala-brotli-07.txt

```diff
@@ -1854,9 +1854,9 @@ Internet-Draft                   Brotli                     October 2015
 
          else
             fill CMAPD[] with zeros
-         read array of prefix codes for literals, HTREEL[]
-         read array of prefix codes for insert-and-copy, HTREEI[]
-         read array of prefix codes for distances, HTREED[]
+         read array of literals prefix codes, HTREEL[]
```

Comment:
One more nit: I think it should be 'literal prefix codes'. No need to put both 'literal' and 'code' in plural. (and similarly for the other two).